packages/content/src/Application/ContentNormalizer/Normalizer/ExcerptNormalizer.php

```diff
@@ -19,6 +19,15 @@ class ExcerptNormalizer implements NormalizerInterface
 {
     public function enhance(object $object, array $normalizedData): array
     {
+        if (!$object instanceof ExcerptInterface) {
+            return $normalizedData;
+        }
+
+        $excerptData = $object->getExcerptData();
+        if (isset($excerptData['excerpt'])) {
+            $normalizedData['excerpt'] = $excerptData['excerpt'];
+        }
```

Comment:
Thik this is not valid?

```suggestion
        $excerptData = $object->getExcerptData();
        $normalizedData['excerpt'] = $excerptData;
```

Reply:
changed in https://github.com/Prokyonn/sulu/pull/7